### ISSUE TITLE
Improving Model Accuracy

### ISSUE THREAD

**Issue by shivranjan0** [shivranjan0/deep_fake_detection-1#62]
Work on improving CNN performance by:

Increasing dataset size

Using data augmentation

Adjusting CNN layers

Tuning hyperparameters like learning rate, epochs, and batch size
